pkgs/top-level/python-packages.nix

```
@@ -5608,6 +5608,8 @@ in {
 
   safe = callPackage ../development/python-modules/safe { };
 
+  sane = callPackage ../development/python-modules/sane { };
```

future proof against a package of the same name being added python-modules
```suggestion
  sane = callPackage ../development/python-modules/sane {
    inherit (pkgs) saneBackends;
  };
```

---

:+1:.